### PR TITLE
Issue 15509 page selector tests

### PR DESCRIPTION
**Pull request by hmoreras**

### CODE REVIEW

File: src/app/view/components/_common/dot-page-selector/dot-page-selector.component.html [hunk 1]
```diff
@@ -32,3 +32,9 @@
         [suggestions]="results.data">
     </p-autoComplete>
 </ng-template>
+
+<dot-icon-button class="dot-page-selector__hint" icon="help_outline" (click)="hint.toggle($event)"></dot-icon-button>
```
Comment (fmontes) [inline src/app/view/components/_common/dot-page-selector/dot-page-selector.component.html]: This should be a component like `field-helper` that receive the html or string to display.

Comment (fmontes) [reply src/app/view/components/_common/dot-page-selector/dot-page-selector.component.html]: And reuse in the other field.

File: src/app/view/components/_common/dot-page-selector/dot-page-selector.component.scss [hunk 2]
```diff
@@ -8,4 +10,11 @@
             white-space: nowrap;
         }
     }
+
+    .dot-page-selector__hint{
```
Comment (fmontes) [inline src/app/view/components/_common/dot-page-selector/dot-page-selector.component.scss]: space in {

File: src/app/view/components/_common/dot-page-selector/dot-page-selector.component.ts [hunk 3]
```diff
@@ -50,7 +50,7 @@ export class DotPageSelectorComponent implements ControlValueAccessor {
         public dotMessageService: DotMessageService
     ) {
         this.dotMessageService
-            .getMessages(['page.selector.no.sites.results', 'page.selector.no.page.results'])
+            .getMessages(['page.selector.no.sites.results', 'page.selector.no.page.results', 'page.selector.hint'])
```
Comment (alfredo-dotcms) [inline src/app/view/components/_common/dot-page-selector/dot-page-selector.component.ts]: `take(1)`

**Pull request merged**